processing/src/main/java/io/druid/segment/incremental/IncrementalIndexAddResult.java

```diff
@@ -44,6 +47,13 @@ public int getRowCount()
     return rowCount;
   }
 
+
+  public long getBytesInMemory()
+  {
+    return bytesInMemory;
+  }
+
```

Comment:
One two many newlines here.

Reply:
Hmm, if I try to put this method in single line, `mvn install` complains `[ERROR] ~/druid/processing/src/main/java/io/druid/segment/incremental/IncrementalIndexAddResult.java:50:34: '{' at column 34 should be on a new line. [LeftCurly]
[ERROR] ~/druid/processing/src/main/java/io/druid/segment/incremental/IncrementalIndexAddResult.java:50:58: '}' at column 58 should be alone on a line. [RightCurly]`